pages/index.js

```diff
@@ -1,40 +1,77 @@
-import Heading from "@/Header";
 import InterviewCard from "@/components/InterviewCard";
 import { interviews } from "@/public/interviews";
 import SearchInput from "@/components/Search/SearchInput";
 import SearchResults from "@/components/Search/SearchResults";
 import { useState } from "react";
 import Footer from "@/components/Footer";
+import styled from "styled-components";
 
+const INTERVIEWS_PER_PAGE = 2;
 export default function HomePage() {
   const [searchTerm, setSearchTerm] = useState("");
-
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalPages = Math.ceil(interviews.length / INTERVIEWS_PER_PAGE);
+  const paginatedInterviews = interviews.slice(
+    (currentPage - 1) * INTERVIEWS_PER_PAGE,
+    currentPage * INTERVIEWS_PER_PAGE
+  );
   function handleSearch(term) {
     setSearchTerm(term);
   }
   return (
     <>
-      <main>
+      <StyledMain>
         <SearchInput searchTerm={searchTerm} onSearch={handleSearch} />
         {searchTerm ? (
           <SearchResults searchTerm={searchTerm} />
         ) : (
-          <ul>
-            {interviews.map((interview) => (
-              <li className="interview-card" key={interview.id}>
-                <InterviewCard
-                  id={interview.id}
-                  title={interview.title}
-                  summary={interview.summary}
-                  text={interview.text}
-                  image={interview.image}
-                />
-              </li>
-            ))}
-          </ul>
+          <>
+            <ul>
+              {paginatedInterviews.map((interview) => (
+                <li className="interview-card" key={interview.id}>
+                  <InterviewCard
+                    id={interview.id}
+                    title={interview.title}
+                    summary={interview.summary}
+                    text={interview.text}
+                    image={interview.image}
+                  />
+                </li>
+              ))}
+            </ul>
+
+            {currentPage < totalPages && (
+              <PageNavBtn onClick={() => setCurrentPage(currentPage + 1)}>
+                Next Page
+              </PageNavBtn>
+            )}
+            {/* Show previous button if we are on any page greater than 1 */}
+            {currentPage > 1 && (
+              <PageNavBtn onClick={() => setCurrentPage(currentPage - 1)}>
+                Previous Page
+              </PageNavBtn>
+            )}
+          </>
         )}
-      </main>
+      </StyledMain>
       <Footer currentPath="/" />
     </>
   );
 }
+
+const StyledMain = styled.main`
+  margin-bottom: 200px;
+`;
+const PageNavBtn = styled.button`
+  width: 110px;
```

Comment:
I think 110px is overall a fixed width that I wouldn't mind too much, but otherwise fixed widths can be somewhat of a struggle to maintain when we're trying to maintain responsive designs.

Reply:
Thank you! noted down and will go through it in the next phase of the project.